go/tasks/plugins/presto/client/presto_status.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	"context"
+	"strings"
+
+	"github.com/lyft/flytestdlib/logger"
+)
+
+type PrestoStatus string
+
+// This type is meant only to encapsulate the response coming from Presto as a type, it is
+// not meant to be stored locally.
+const (
+	PrestoStatusUnknown   PrestoStatus = "UNKNOWN"
+	PrestoStatusQueued    PrestoStatus = "QUEUED"
+	PrestoStatusRunning   PrestoStatus = "RUNNING"
+	PrestoStatusFinished  PrestoStatus = "FINISHED"
+	PrestoStatusFailed    PrestoStatus = "FAILED"
+	PrestoStatusCancelled PrestoStatus = "CANCELLED"
+)
+
+var PrestoStatuses = map[PrestoStatus]struct{}{
```

Comment:
I could swear I left this feedback but I can't see it... 
Can you see if you can use this [instead](https://github.com/lyft/flyteplugins/blob/dc076e467ed5fbf34fce3506db676a8556376463/go/tasks/pluginmachinery/core/transition.go#L5).

Reply:
@EngHabu I think  this was your comment: https://github.com/lyft/flyteplugins/pull/69#discussion_r391834458
I replied saying that it doesn't seem like it's supported for strings.